concepts/05 UI Components/HtmlEditor/00 Getting Started with HtmlEditor/04 Customize the Toolbar.md

```diff
@@ -0,0 +1,258 @@
+If you need to customize HtmlEditor's toolbar, pass an array of [items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) to the [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/). There is a number of [predefined items](/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Predefined_Items/) available for customization. Some of the items, such as font size, support multiple values. In these cases use the [formatName](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/#formatName) property to define item name and the [formatValue](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/#formatValues) property to assign an array of values you need to make available. Set the [multiline](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/#multiline) property to **true** to display the toolbar items on multiple lines.
```

Comment:
- You start with customizationn but what does the toolbar contain?
- "There is a number of [predefined items](/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Predefined_Items/) available for customization." - predefined items of different types are customized differently. Moreover, your can add a custom item, which is also a part of toolbar customization.
- "use the [formatName](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/#formatName) property to define item name and the [formatValue](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/#formatValues) property" - these are v20.2 names of these properties. We are writing the tutorial for v21.1.